Hyper/Scene.cs

```diff
@@ -39,66 +43,110 @@ internal class Scene : IInputSubscriber
 
     private readonly ScalarFieldGenerator _scalarFieldGenerator;
 
+    private readonly int _chunksPerSide = 2;
+
+    private readonly SimulationManager<NarrowPhaseCallbacks, PoseIntegratorCallbacks> _simulationManager;
+
+    private readonly CollidableProperty<SimulationProperties> _properties;
+
+    private readonly SimpleCar _simpleCar;
+
+    private readonly Vector3 _carInitialPosition;
```

Comment:
these 3 should have = null!

Reply:
not sure why, they are diligently initialized in the constructor